rs/nns/governance/BUILD.bazel

```diff
@@ -77,19 +77,23 @@ BASE_DEPENDENCIES = [
 # Each target declared in this file may choose either these (release-ready)
 # dependencies (`DEPENDENCIES`), or `DEPENDENCIES_WITH_TEST_FEATURES` feature previews.
 DEPENDENCIES = BASE_DEPENDENCIES + [
+    "//rs/nervous_system/tla",
```

Comment:
Rename this to `//rs/nervous_system/tla:tla-noop` and the other one (`--test_feature` to `tla-enabled`)